health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/services/PaymentsInitialization.js

```
@@ -0,0 +1,168 @@
+  // Process the root boundaries
+  tenantBoundary.forEach((boundary) => traverse(boundary, 1));
+
+  return order;
+};
+
+const initializePaymentsModule = async ({ tenantId }) => {
+
+  const projectContextPath = window?.globalConfigs?.getConfig("PROJECT_CONTEXT_PATH") || "health-project";
+  const individualContextPath = window?.globalConfigs?.getConfig("INDIVIDUAL_CONTEXT_PATH") || "health-individual";
+
+  let user = Digit?.SessionStorage.get("User");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'user' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)